avalara_salestax/tests/test_invoice_validation.py

```diff
@@ -7,7 +7,7 @@ class TestInvoiceValidationSetUp(TransactionCase):
     def setUp(self):
         super(TestInvoiceValidationSetUp, self).setUp()
         config = self.env['avalara.salestax'].search([
-            ('company_code', 'not like', 'Contact Avalara')], limit=1)
+            ('company_code', 'like', 'Contact Avalara')], limit=1)
```

Comment:
Nope ;-)

Reply:
But given that I removed the dummy config with this name, you can remove the filter and just search for any config.